homeassistant/components/kodi/media_player.py

```diff
@@ -639,7 +639,36 @@ async def async_play_media(self, media_type, media_id, **kwargs):
         media_type_lower = media_type.lower()
 
         if media_type_lower == MEDIA_TYPE_CHANNEL:
-            await self._kodi.play_channel(int(media_id))
+            _LOGGER.debug("Searching channel...")
```

Comment:
Let's keep this method simple, and move this functionality to its own method, like it is for searching artists/albums.